dashboard/test/helpers/workshop_survey_results_helper_test.rb

```diff
@@ -143,4 +209,91 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
       summarize_workshop_surveys(workshops: [local_workshop, teachercon])
     end
   end
+
+  test 'daily survey get_question_for_forms gets workshop questions and substitutes question texts' do
+    CDO.stubs(:jotform_forms).returns({'local' => {'day_0' => PRE_SURVEY_STUBBED_FORM_ID}})
```

Comment:
If we use `expects` instead of `stubs` we'll get an additional validation that this is actually called.